util/openapi-generator.ts

```diff
@@ -1,3 +1,9 @@
+import * as dotenv from 'dotenv';
+
+// First load default.env to prevent envSchema from throwing errors
+const defaultParsed = dotenv.config({ path: `${__dirname}/default.env` }).parsed;
+dotenv.populate(process.env as Record<string, string>, defaultParsed as Record<string, string>);
```

Comment:
This is indicative of a problem with our ENV init approach. Our modules have side-effects -- in this case `src/env.ts` has top-level evaluations to init the `ENV` object. 

These lines setup some env var defaults to prevent that module from throwing errors. A better approach would be to remove side-effects from modules, for example make ENV a getter that only evals on demand.